.github/workflows/ci.yml

```diff
@@ -580,3 +580,16 @@ jobs:
       #   # Don't run on forks, since secrets won't be available, and command will fail
       #   if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository && github.ref != 'refs/heads/main'
       #   run: npx vercel deploy packages/docs/build -t ${{ secrets.VERCEL_TOKEN }} --name typescript --scope temporal --yes
+
+  omes-image-build:
```

Comment:
Do we want that even for PR branches? Are we expected for this to completes before merging a PR to main?

Reply:
Yes. The idea is to run these per-pr. If it's too slow we can make it optional.